samples-android/ButtonClicker/jni/ButtonClickerNativeActivity.h

```diff
@@ -147,13 +147,13 @@ class Engine : public gpg::IRealTimeEventListener {
                             // in OnRoomStatusChanged()
   int32_t score_counter_;   // Score counter of local player
   bool playing_;            // Am I playing a game?
-  bool authorizing_;        // Am I signing in to gpg service?
   std::string self_id_;     // Local player's ID
   double start_time_;       // Game start time
 
   // synchronization primitive to synchronize
   // UIThread, Timer thread and gpg callback thread
   mutable std::mutex mutex_;
+  mutable std::mutex startup_mutex_;
```

Comment:
this one is just for start up code: taking conservative way not to make a big change to the existing code.